runtime/src/accounts_index.rs

```diff
@@ -23,6 +30,26 @@ pub type RefCount = u64;
 pub type AccountMap<K, V> = BTreeMap<K, V>;
 
 type AccountMapEntry<T> = Arc<AccountMapEntryInner<T>>;
+type SecondaryIndexEntry = DashMap<Pubkey, RwLock<HashSet<Slot>>>;
+
+enum ScanTypes<R: RangeBounds<Pubkey>> {
+    Standard(Option<R>),
```

Comment:
nits: rename to `Full{Scan}`. well, I think it's clearer than the Standard of Scan

Reply:
Renamed to UnIndexed, because sometimes it's a range scan, not necessarily a full scan